.travis.yml

```diff
@@ -19,6 +19,7 @@ install:
 script:
   - flutter analyze
   - flutter test --coverage
+  - dartdoc
```

Comment:
This will fix this?
```
Maintenance issues and suggestions

Make sure dartdoc successfully runs on your package's source files. (-10 points)

Reply:
This one should make sure it passes the check. It looks it runs properly locally on my machine.